Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Description: Dockerfile for the Helsinki tileserver-gl image
+# Build: docker build -t helsinki/tileserver-gl:latest .
+# Run: docker run --rm -it -v /local/data/mbtiles:/data/mbtiles -p 8080:8080 helsinki/tileserver-gl:latest
+
+# local development base image path
+# FROM maptiler/tileserver-gl:v5.1.3
+FROM container-registry.platta-net.hel.fi/devops/maptiler/tileserver-gl:v5.1.3
+
+USER root:root
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends --no-install-suggests \
+      curl gettext-base && \
+    apt-get -y --purge autoremove && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV SOURCES_OPENMAPTILES_URL "mbtiles://helsinki.mbtiles"
```

Comment:
Can this be renamed to `world.mbtiles` or similar because `helsinki.mbtiles` refers to old usage and nowadays dataset covers the whole world.

Reply:
Yes. I will open a separate PR for this once I've merged this and updated azure to use GH as the main source for the pipeline.  :+1: